Podfile.lock

```diff
@@ -18,22 +20,128 @@ PODS:
     - TomTomSDKBindingFrameworkLoggingInternal (= 0.22.1)
     - TomTomSDKBindingFrameworkSecurityInternal (= 0.22.1)
   - TomTomSDKBindingNavigationTextGenerationInternal (0.22.1)
+  - TomTomSDKBindingNavigationTileStoreAccessInternal (0.22.1):
+    - TomTomSDKBindingFrameworkLoggingInternal (= 0.22.1)
+  - TomTomSDKBindingNavigationTilingInternal (0.22.1)
+  - TomTomSDKBindingNDSSQLiteInternal (0.22.1)
+  - TomTomSDKBindingNDSStoreAccessInternal (0.22.1):
+    - TomTomSDKBindingNDSSQLiteInternal (= 0.22.1)
+  - TomTomSDKBindingOnboardMapUpdateInternal (0.22.1):
+    - TomTomSDKBindingNDSSQLiteInternal (= 0.22.1)
+    - TomTomSDKBindingNDSStoreAccessInternal (= 0.22.1)
+  - TomTomSDKBindingOnboardOpenLRInternal (0.22.1):
+    - TomTomSDKBindingFrameworkLoggingInternal (= 0.22.1)
+    - TomTomSDKBindingNDSSQLiteInternal (= 0.22.1)
+    - TomTomSDKBindingNDSStoreAccessInternal (= 0.22.1)
+  - TomTomSDKBindingRouteProjectionInternal (0.22.1)
+  - TomTomSDKBindingVehicleHorizonInternal (0.22.1)
+  - TomTomSDKBindingVehicleHorizonNDSMapInternal (0.22.1):
+    - TomTomSDKBindingNDSSQLiteInternal (= 0.22.1)
+    - TomTomSDKBindingNDSStoreAccessInternal (= 0.22.1)
   - TomTomSDKCommon (0.22.1):
     - TomTomSDKBindingFrameworkLoggingInternal (= 0.22.1)
   - TomTomSDKCommonUI (0.22.1):
     - TomTomSDKCommon (= 0.22.1)
+  - TomTomSDKDataManagement (0.22.1):
+    - TomTomSDKCommon (= 0.22.1)
+  - TomTomSDKDataManagementOffline (0.22.1):
```

Comment:
Could you execute `pod install` once again after changing `Podfile` to exclude offline dependencies?